compact/tracker/mpgd_dirc.xml

```diff
@@ -0,0 +1,185 @@
+    <constant name="MPGDDIRCFrame_width"            value="20*mm"/>
+    <constant name="MPGDDIRCFrame_thickness"        value="7*mm"/>
+
+    <comment> module constants </comment>	
+    <constant name="MPGDDIRCModule_count"                   value="12" />
+    <constant name="MPGDDIRCModule_allowed_space"           value="2*cm"/>
+    <constant name="MPGDDIRCModule_rmin"                    value="MPGDDIRC_rmin"/>
+    <constant name="MPGDDIRCModule_rmax"                    value="MPGDDIRCModule_rmin + MPGDDIRCModule_allowed_space" />
+    <constant name="MPGDDIRCModule_zmin1"                   value="145.0*cm"/>
+    <constant name="MPGDDIRCModule_zmin2"                   value="197.0*cm"/>
+    <constant name="MPGDDIRCModule_width"                   value="36.2*cm"/>
+    <constant name="MPGDDIRCModule_length"                  value="0.5*(MPGDDIRCModule_zmin1 + MPGDDIRCModule_zmin2)"/>
+    <constant name="MPGDDIRCModule_offset"                  value="(MPGDDIRCModule_zmin1 - MPGDDIRCModule_zmin2)/2.0"/>
+
+
+
+    <comment> layer parameters </comment>	
+    <constant name="MPGDDIRCWindow_thickness"	          value="50*um"/>
+    <constant name="MPGDDIRCWindowGap_thickness"          value="2*mm"/>
+    <constant name="MPGDDIRCDriftGap_thickness"           value="3*mm"/>
+    <constant name="MPGDDIRCFoilCu_thickness"	          value="5*um"/>
+    <constant name="MPGDDIRCReadOutElectrode_thickness"   value="10*um"/>
+    <constant name="MPGDDIRCFoilKapton_thickness"         value="50*um"/>
+    <constant name="MPGDDIRCReadOutNomex_thickness"       value="50*um"/>
+    <constant name="MPGDDIRCReadOutKapton_thickness"      value="50*um"/>
+    <constant name="MPGDDIRCPCB_thickness"                value="3*mm"/>
+
+    <comment> Thickness values </comment>	
+    <constant name="MPGDDIRCCathode_thickness"          value="MPGDDIRCFoilKapton_thickness + MPGDDIRCFoilCu_thickness"/>
+    <constant name="MPGDDIRCFoil_thickness"             value="MPGDDIRCFoilKapton_thickness + MPGDDIRCFoilCu_thickness"/>
+    <constant name="MPGDDIRCReadOut_thickness"          value="MPGDDIRCReadOutNomex_thickness + MPGDDIRCReadOutElectrode_thickness + MPGDDIRCReadOutKapton_thickness"/> 
+    <constant name="MPGDDIRCModule_thickness"           value="MPGDDIRCWindow_thickness + MPGDDIRCCathode_thickness + 
+	                                                         MPGDDIRCFoil_thickness + MPGDDIRCReadOut_thickness + 
+	                                                         MPGDDIRCPCB_thickness + MPGDDIRCFrame_thickness" />
+  </define>
+
+
+  <materials>
+  </materials>
+
+  <limits>
+  </limits>
+
+  <regions>
+  </regions>
+
+  <display>
+  </display>
+
+   <detectors>
+   <detector id="MPGDDIRC_0_ID" name="MPGDDIRC_0" type="epic_MPGDDIRC" readout="MPGDDIRCHits" vis="TrackerVis">
+	    <dimensions width="MPGDDIRCModule_width" length="MPGDDIRCModule_length" height="MPGDDIRCModule_allowed_space" />
+	    <position x="0" y="0" z="MPGDDIRCModule_offset" />
```

Comment:
There are still tabs :-)

Reply:
`xmllint --format compact/tracker/mpgd_dirc.xml` or `sed -i `s/\t/  /g' compact/tracker/mpgd_dirc.xml` but some postprocessing would still be needed.

Reply:
Thanks. I think the tabs are gone now😬